source/documentation/fuseki2/fuseki-data-services.md

```diff
@@ -17,7 +17,7 @@ update
 GSP including direct naming for GSP
 
 in addition:
-
+fuse
```

Comment:
fuseki??? Or was it an accidental copy-and-paste?

Reply:
"fuseki-data-services" is linked to the documentation and it looks like it has largely been overtaken. The bit about active/offline isn't accurate. I'll delete "fuseki-data-services".